trajectory_publisher/src/polynomialtrajectory.cpp

```diff
@@ -128,6 +128,10 @@ Eigen::VectorXd polynomialtrajectory::getCoefficients(int dim) {
   }
 }
 
+float polynomialtrajectory::getYaw(double time) {
+  float yaw = 0.0;
```

Comment:
Where does `yaw=0.0` come from? This needs to configurable, and probably belongs to the ignore yaw case